index.html

```diff
@@ -115,6 +115,29 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
         
+        <div class="card">
+          <p class="name">Ahammed Imthias</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/" target="https://twitter.com/LeviAckerman__l">Levi_ackerman_l</a>
+          </p>
+          <p class="about">about you</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="_blank" title="First Resource">Resource 1</a>
```

Comment:
add 3 resources that can help felow developers with a link in place of `#`, a suitable title by replacing `First Resource` and the resource name in the place where `Resource 1` is written.
```suggestion
                <a href="#" target="_blank" title="First Resource">Resource 1</a>
```